src/table.js

```diff
@@ -324,13 +323,9 @@ export const selectTable = tr => {
 // ```
 export const emptyCell = (cell, schema) => tr => {
   if (cell) {
-    const content = tableNodeTypes(schema).cell.createAndFill().content;
+    const { content } = tableNodeTypes(schema).cell.createAndFill();
     if (!cell.node.content.eq(content)) {
-      tr.replaceWith(
-        cell.pos,
-        cell.pos + cell.node.nodeSize - 1,
-        new Slice(content, 0, 0)
-      );
+      tr.replaceWith(cell.pos + 1, cell.pos + cell.node.nodeSize, content);
```

Comment:
Why did you remove the `-1` on the second parameter? because of the `nodeSize` works like length and the `replaceWith` needs a index.

Reply:
and I don't think the `cell.pos+1` will work too, could you write tests for it?

Reply:
That's the beauty: tests are already in place, and they fail unless you offset these positions by 1. I think we formerly targeted the tableCell, now it targets the tableCell's content.